### PR TITLE
feat button & alert added

### PR DESCRIPTION
**Pull request by sagar455**
## button and alert
image

### CODE REVIEW

**Review by Abhijeet43 — CHANGES_REQUESTED** [review]
Hey, @sagar455 try to align icon and text together in Icon buttons

**Comment by sagar455** [comment]
> Hey, @sagar455 try to align icon and text together in Icon buttons

Sure i will fix it now

**Review by Abhijeet43 — APPROVED** [review]
Looks Good Now!!!!!

File: alerts&buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    width:8rem;
+    height:2.5rem;
+    border: solid 1px;
+    border-radius: 0.5rem;
+    cursor:pointer;
+}
+
+.btn-icon{
+    display:flex;
+    flex-direction: row;
```
Comment (kart-c) [inline alerts&buttons/buttons.css:32]: You can remove this, as it's default property

**Review by kart-c — APPROVED** [review]
Hey Sagar, you can remove the html comments as the text inside is pretty self explainatory

File: alerts&buttons/buttons.css (hunk 1 above)
Comment (sagar455) [reply alerts&buttons/buttons.css:32]: Ok i will remove it

**Pull request merged**